presto-main/src/test/java/io/prestosql/sql/planner/TestPlanMatchingFramework.java

```diff
@@ -257,4 +260,11 @@ public void testProjectLimitsScope()
                         project(ImmutableMap.of("EXPRESSION", expression("CAST(1 AS bigint) + ORDERKEY")),
                                 tableScan("lineitem", ImmutableMap.of("ORDERKEY", "orderkey")))));
     }
+
+    private Session disableCbo()
```

Comment:
rename to `noJoinReordering` (as in https://github.com/prestosql/presto/blob/90833ad2484a2206ce7f572eb911611be9cacf0d/presto-testing/src/main/java/io/prestosql/testing/AbstractTestJoinQueries.java#L2278-L2284)